tests/integration/bundle/test_tls_bundle.py

```diff
@@ -20,7 +24,7 @@
 logger = logging.getLogger(__name__)
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
-FINOS_WALTZ = "finos-waltz"
+FINOS_WALTZ = "finos-waltz-k8s"
```

Comment:
I tried to switch this test to use Mattermost, since now pgbouncer is also using TLS, but the pod created by it gets stuck in a crash loop on github runners.